common.js

```diff
@@ -116,6 +116,17 @@ var ccg = {
       status: "Internet-Draft",
       publisher: "IETF"
     },
+    "BASE58": {
+      title: "The Base58 Encoding Scheme",
+      date: "December 2019",
+      href: "https://tools.ietf.org/html/draft-msporny-base58",
+      authors: [
+        "Satoshi Nakamoto",
```

Comment:
Delete the Satoshi Nakamoto author from this Internet Draft.  The IETF requires IPR declarations from all authors and unless you can convince Satoshi to make his IPR declaration, this draft will not be advanced to RFC status with him as an author.

Reply:
Done.